fe/projects/AspNetCore/src/shared/components/OrderCatalog/Catalog.js

```diff
@@ -226,7 +226,7 @@ function Catalog(props) {
                                                         {props.table.actions.map((actionItem, index) => {
                                                             if (actionItem.isEdit) return (
                                                                 <Tooltip title={props.editLabel} aria-label={props.editLabel} key={index}>
-                                                                    <Fab href={props.editUrl + "/" + item.id} size="small" color="secondary">
+                                                                    <Fab href={props.editUrl + "/" + item.id + "?" + QueryStringSerializer.serialize({ searchTerm: searchTerm })} size="small" color="secondary">
```

Comment:
**suggestion:** I think setting searchTerm could be simplified with ?searchTerm={searchTerm} or searchTerm ? "?searchTerm={searchTerm} : ""